### PR TITLE
docs: standardize titles for celestia docs & fix linting

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

This PR standardizes the titles that are being used in https://github.com/celestiaorg/docs/pull/668 and resolves [lint errors](https://github.com/celestiaorg/docs/actions/runs/4737250543/jobs/8409824190?pr=668) that occurred when the markdown was imported from the `orchestrator-relayer` repo.

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/keys.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Keys
+# Keys subcommand
```
Comment (SweeXordious) [inline docs/keys.md:1]:
```suggestion
# Keys management
```

Comment (SweeXordious) [inline docs/keys.md:1]:
```suggestion
# Keys manager
```

Comment (SweeXordious) [reply docs/keys.md:1]: I think these sound better, what do you think?

**Review by SweeXordious — COMMENTED** [review]
Thanks a lot for this

File: docs/keys.md (hunk 1 above)
Comment (jcstein) [reply docs/keys.md:1]: i like it. what do you think of `Key management`?

Comment (jcstein) [reply docs/keys.md:1]: agreed! I'm leaning toward `Key management`

Comment (SweeXordious) [reply docs/keys.md:1]: Sounds good :rocket:

Comment (SweeXordious) [reply docs/keys.md:1]: Sounds good :rocket:

**Review by SweeXordious — APPROVED** [review]

**Pull request merged**